sencha-workspace/packages/slate-cbl/src/view/student/skill/OverviewWindow.js

```diff
@@ -0,0 +1,87 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.student.skill.OverviewWindow', {
+    extend: 'Ext.window.Window',
+    xtype: 'slate-cbl-student-skill-overviewwindow',
+    requires: [
+        'Slate.cbl.view.student.skill.OverviewWindowController',
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    controller: 'slate-cbl-student-skill-overviewwindow',
+
+    config: {
+        student: null,
+        competency: null,
+        skill: null
+    },
+
+    title: 'Standard Overview',
+    width: 700,
+    minWidth: 700,
+    shadow: 'frame',
+    center: true,
+    fixed: true,
```

Comment:
again I'm concerned with what will happen when the overviewWindow is bigger than the browser window, doing this will require an inner scrolling solution.

My initial design principal as I encountered these issues was to avoid using inner scrolling in any detail windows so that users could treat the page as one large canvas. This might make more sense on the teacher end than it does on the student end though. While the window layout in its previous state had less-than-ideal UX, it didn't have the potential to make information inaccessible like this seems to. I would comment this out for now until we can explore a more thorough solution to bigger-that-viewport windows, which i could see either going the direction of modal with inner scrolling or growing with some smarter placement/resize sugar